rundeckapp/grails-app/controllers/rundeck/controllers/ExecutionController.groovy

```diff
@@ -2133,7 +2129,7 @@ setTimeout(function(){
      * API: /api/5/executions query interface, deprecated since v14
```

Comment:
I think maybe this entire API endpoint mapping in UrlMappings should be removed, but I will have to review that